src/main/java/org/zalando/logbook/Named.java

```diff
@@ -0,0 +1,30 @@
+package org.zalando.logbook;
+
+/*
+ * #%L
+ * Logbook
+ * %%
+ * Copyright (C) 2015 Zalando SE
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+interface Named {
+
+    default String getName() {
+        // TODO this will use the system hash (hopefully), is this correct?
```

Comment:
As long as the class implementing this interface does not override `toString`. Maybe you want something like

    getClass().getName() + "@" + Integer.toHexString(System.identityHashCode(this))

Again, I would prefer to put this in a method of `OnceFilter`, this should really be a hidden implementation detail of the filter.